inference/core/workflows/core_steps/flow_control/continue_if.py

```diff
@@ -38,7 +39,7 @@ class BlockManifest(WorkflowBlockManifest):
             "block_type": "flow_control",
         }
     )
-    type: Literal["Condition"]
```

Comment:
Would old `Condition` blocks still work if we left them in here (but deprecated & hidden from UI)?

Reply:
We could do optional else statement - remaining Condition functionalities

Reply:
@hansent @EmilyGavrilenko @grzegorz-roboflow - please state ur opinions on how we shape condition block

Reply:
If `Condition` can have optional (or just default `no-op`/ `stop-execution` value), I have slight preference for Condition with `if` and `else` as opposed to just `ContinueIf`.  I think having both would be OK.   

I think just having `ContinueIf` is fine for shipping this version and we can add additional block with else later if users ask for it / are confused by ContinueIf / constantly run into wanting an else and having to define two blocks for it.